src/Microsoft.Framework.Runtime/NuGet/Authoring/Manifest.cs

```diff
@@ -57,15 +60,30 @@ public void Save(Stream stream, bool validate)
         public void Save(Stream stream, bool validate, int minimumManifestVersion)
         {
             int version = Math.Max(minimumManifestVersion, ManifestVersionUtility.GetManifestVersion(Metadata));
-            string schemaNamespace = ManifestSchemaUtility.GetSchemaNamespace(version);
+            var schemaNamespace = (XNamespace)ManifestSchemaUtility.GetSchemaNamespace(version);
+
+            var root = new XElement(schemaNamespace + "package");
+
+            var metadataElement = Metadata.ToXElement(schemaNamespace);
+            if (metadataElement == null)
+            {
+                throw new InvalidOperationException("Failed to serialize manifest metadata to xml.");
+            }
+            root.Add(metadataElement);
+
+            if (Files != null && Files.Count != 0)
+            {
+                var filesElement = new XElement(schemaNamespace + "files");
+                foreach (var file in Files)
+                {
+                    filesElement.Add(file.ToXElement(schemaNamespace));
+                }
 
-            // Define the namespaces to use when serializing
-            var ns = new XmlSerializerNamespaces();
-            ns.Add("", schemaNamespace);
+                root.Add(filesElement);
+            }
```

Comment:
Consider being more fluent - you should be able to create an Xml document in one go preserving visually the document structure. Something like this:

```
var metadataElement = Metadata.ToXElement(schemaNamespace);
if (metadataElement == null)
{
    throw new InvalidOperationException("Failed to serialize manifest metadata to xml.");
}

new XDocument(
    new XElement(
        schemaNamespace + "package", 
            metadataElement,
            Files.Any() ? 
            new XElement(
                schemaNamespace + files, 
                    Files.Select(f => f.ToXElement)))).Save(stream);
```

(Yeah, not wanting empty "files" element breaks the pattern a little bit - note you can pass null so it still can be inline)